modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-commit.adoc

```diff
@@ -0,0 +1,43 @@
+= Use GCP Commits
+:description: Subscribe to Redpanda in Google Cloud Marketplace with discounted commitment billing.
+
+With a usage-based billing commitment, you sign up for a minimum spend amount and get a discount on your committed usage. This allows you to subscribe to Redpanda Cloud through Google Cloud Marketplace to quickly provision Redpanda Dedicated clusters. 
+
+Commitments are minimums. If you use more than your committed amount, you can continue using Redpanda Cloud without interruption. You will be charged at your discounted rate for usage beyond the committed amount until the end of your commitment term. 
+
```

Comment:
ADD:

If your usage is below your commitment minimum, you are still expected to pay the minimum but a rollover occurs for the unused amount which is set to expire for the end of contract.

(Feel free to wordsmith this some)